bin/plant-beanstalk.js

```diff
@@ -19,3 +20,8 @@ const dockerrun = {
 const output = JSON.stringify(dockerrun, null, 2)
 
 fs.writeFileSync(path.resolve(__dirname, '../Dockerrun.aws.json'), output)
+
+const zip = new AdmZip()
+zip.addLocalFile(`${__dirname}/../Dockerrun.aws.json`)
```

Comment:
Are you not worried about any of this erroring?

Reply:
Only a little - it only happens during the circle CI build in order to generate the deployment artifacts, so if misconfigured the build would fail, but it's not going to be a runtime issue